pages/index.tsx

```diff
@@ -1,62 +1,105 @@
 import Head from 'next/head';
+import { Formik, Form } from 'formik';
+import FormField from '../components/FormField';
+import FormSelect, { FormSelectItem } from '../components/FormSelect';
+import Button from '@material-ui/core/Button';
+import { Typography } from '@material-ui/core';
+
+interface FormValues {
+  name: string;
+  type: string;
+  theme: string;
+}
+
+const initialValues: FormValues = {
+  name: '',
+  type: '',
+  theme: '',
+};
+
+const positionItems: FormSelectItem[] = [
+  {
+    label: 'Corporate',
+    value: 'corporate',
+  },
+  {
+    label: 'AI',
+    value: 'ai',
+  },
+  {
+    label: 'Technology',
+    value: 'technology',
+  },
+  {
+    label: 'E-Commerce',
+    value: 'e_commerce',
+  },
+];
+
+const themeItems: FormSelectItem[] = [
+  {
+    label: 'Dark Theme',
+    value: 'dark',
+  },
+  {
+    label: 'Light Theme',
+    value: 'light',
+  },
+];
+
+const Home: React.FC = () => {
+  const handleSubmit = (values: FormValues): void => {
+    // Make API Call with JSON.stringify(values);
```

Comment:
We will just be rerouting to /home/:seed so you will need to generate a random number to be used as the seed and add the options in some way, either `/home/94829483?type=e_commerce` or  `/home/e_commerce-94829483` or you could encode the type to make it less obvious what we're doing eg. `/home/MVPWG33NNVSXEY3F-94829483`